include/RAJA/internal/type_traits.hxx

```diff
@@ -97,11 +97,55 @@ struct function_traits<R (T::*)(Args...) const> {
 
 // extract 0-based argument type
 template <size_t I, typename Fn>
-using extract_arg = std::tuple_element<I, typename function_traits<Fn>::argument_types>;
+using extract_arg =
+    std::tuple_element<I, typename function_traits<Fn>::argument_types>;
 
 template <size_t I, typename Fn>
 using extract_arg_t = typename extract_arg<I, Fn>::type;
 
+template <typename Exec>
+struct is_exec_policy
+  : public std::integral_constant<
+      bool, std::is_base_of<RAJA::PolicyBase, Exec>::value> {
```

Comment:
Hmm... After looking they don't, but they either should or the idea of PolicyBase should go away